authentication/wildfly-mods/arquillian.xml

```diff
@@ -0,0 +1,13 @@
+<arquillian xmlns="http://jboss.org/schema/arquillian"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="
+        http://jboss.org/schema/arquillian
+        http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+    <container qualifier="default" default="true">
+        <configuration>
+            <property name="jbossHome">${test.wildfly.home}</property>
+            <property name="javaVmArguments">${debugJvmArgs}</property>
+            <property name="serverName">wildfly</property>
```

Comment:
Definitely not critical, but there is no `serverName` property in the configuration. This could safely be removed.